bin/upload.sh

```diff
@@ -0,0 +1,17 @@
+#!/bin/sh
+
```

Comment:
We should test here `$GDRIVE_REFRESH_TOKEN` and `$GDRIVE_DIR`. If they are undefined, exit this script.